content/general/concepts/software-development-life-cycle/terms/iterative/iterative.md

```diff
@@ -0,0 +1,42 @@
+
+The origins of the iterative approach can be traced back to the 1950s and 1960s when software development was in its infancy. However, it was not until the 1980s that the iterative approach gained wider acceptance and became a popular methodology for software development. At that time, software development was viewed as a linear process that involved several distinct stages, such as requirements gathering, design, coding, testing, and maintenance. However, as software projects became larger and more complex, it became clear that this linear approach was inadequate. In response, software developers began to experiment with iterative models that emphasized incremental progress and continuous feedback. 
+  
+## Current Usage  
+
+Today, the iterative approach is widely used in software development and is seen as an effective way to manage complex projects that involve multiple stakeholders, changing requirements, and evolving technologies. It is particularly well-suited for agile development methodologies that emphasize collaboration, flexibility, and adaptability.
+  
+## Advantages  
+  
+Some of the benefits of the iterative approach to the SDLC include:
+
+- Improved flexibility: The iterative approach allows developers to respond quickly to changing requirements and adapt to new technologies as they emerge.
+- Increased collaboration: The iterative approach emphasizes collaboration between developers, customers, and other stakeholders, which can lead to better communication and a more effective development process.
+- Better risk management: The iterative approach allows developers to identify and mitigate potential risks early in the development process, reducing the likelihood of costly errors later on.
+  
+## Disadvantages
+  
+- Increased complexity: The iterative approach can make the development process more complex as it involves frequent changes and modifications to the requirements and design. This can lead to confusion and difficulties in managing the project.
+- Costly: Iterative development can be more costly than traditional development methods due to the need for frequent testing and rework. Additionally, the need for continuous feedback and collaboration can increase project overheads.
+- Time-consuming: The iterative approach can be time-consuming due to the need for frequent testing and rework. This can lead to delays in the project timeline and increased time to market.
+- Can be difficult to manage: Managing iterative development can be challenging, especially when dealing with large teams and complex projects. It can be difficult to keep track of changes and ensure that everyone is working towards the same goal.
+- Requires experienced team members: The success of the iterative approach depends on having experienced team members who can effectively manage the development process, provide feedback, and make decisions quickly. This can be a challenge for organizations that lack the necessary talent or resources.
```

Comment:
```suggestion
- Requires experienced team members: The success of the Iterative model depends on having experienced team members who can effectively manage the development process, provide feedback, and make decisions quickly. This can be a challenge for organizations that lack the necessary talent or resources.
```